app/services/oneoffs/remove_fees_from_ecf_contracts.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Oneoffs
+  class RemoveFeesFromECFContracts
+    attr_reader :cohort_year, :from_date
+
+    def initialize(cohort_year:, from_date:)
+      @cohort_year = cohort_year
+      @from_date = from_date.to_date
+    end
+
+    def call
+      statements.each do |statement|
```

Comment:
could we avoid going through each statement, and instead going through `CallOffContract.where(cohort: cohort_2021)` as this avoids accidentally creating too many contracts? (see suggested approach in script in ticket)